src/components/webview/SourceControlPanelProvider.ts

```diff
@@ -209,8 +211,33 @@ export class SourceControlPanelProvider {
 							return;
 						}
 
-						// const { title, body } = message.data;
+						const { title, body } = message.data;
 						// call API to create Github Issue
+						const codemodRegistryRepo =
+							'https://github.com/intuita-inc/codemod-registry';
+						const { data } = await axios.post(
+							'https://telemetry.intuita.io/sourceControl/github/issues',
+							{
+								title,
+								body,
+								userId: storedClerkToken,
```

Comment:
hmm, is it a token, or an user ID? this shouldn't be confused.

Reply:
token. will change in studio backend

Reply:
https://github.com/intuita-inc/studio-backend/pull/46

Reply:
renamed it to `accessToken`. Ref: https://github.com/intuita-inc/studio-backend/pull/47